<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QWidget" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 218, 185)
};</string>
  </property>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>monthly patment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_over">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>overpayment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>total payout</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>precent</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>period</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>sum credit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="text">
    <string>type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>580</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
border: 1px solid gray;
};</string>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="label_precent">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="label_sum_credit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="label_period">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: rgb(255, 192, 203);
color: rgb(255, 105, 180);
};</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
